README.md

```diff
@@ -43,15 +43,14 @@ API().run_workflow(input_json='myrun.json')
 ---
 Note: Starting `0.8.2`, Tibanna supports local CWL/WDL files as well as shell commands and Snakemake workflows.
 
-Note2: Python2.7 support is deprecated now (`0.7.0` or after)! Please switch to Python3.6!
+Note 2: As of Tibanna version `2.0.0`, Python 3.6 is deprecated. Please switch to Python 3.8!
```

Comment:
3.6 is not deprecated. This is an incompatible change to make 3.6 unsupported.

Deprecation means "fully supported during a transition period, but something you are advised to move out of". In this case, with no (apparent) notice, you're saying "it went away". That's not deprecation.
```suggestion
Note 2: As of Tibanna version `2.0.0`, Python 3.6 is no longer supported. Please switch to Python 3.8! Python 3.7 is also supported as a fallback, but please prefer 3.8 if you can.
```